app/models/version.rb

```diff
@@ -1,29 +1,35 @@
-  class Version
-    attr_reader :key, :prefix, :photo
+class Version
+  attr_reader :key, :prefix, :photo
 
-    def initialize(photo, key, prefix)
-      @key = key
-      @prefix = prefix
-      @photo = photo
-    end
+  def initialize(photo, key, prefix)
+    @key = key
+    @prefix = prefix
+    @photo = photo
+  end
 
-    def adjust(image)
-      fail "Please override with version specific behaviours"
-    end
+  def adjust(image)
```

Comment:
Unused method argument - `image`. If it's necessary, use `_` or `_image` as an argument name to indicate that it won't be used. You can also write as `adjust(*)` if you want the method to accept any arguments but don't care about them.